anyway/widgets/all_locations_widgets/accident_count_by_day_night_widget.py

```diff
@@ -21,20 +21,32 @@ def __init__(self, request_params: RequestParams):
         )
 
     def generate_items(self) -> None:
-        self.items = get_accidents_stats(
+        all_accident_day_night_count = get_accidents_stats(
             table_obj=AccidentMarkerView,
             filters=self.request_params.location_info,
-            group_by="day_night_hebrew",
-            count="day_night_hebrew",
+            group_by="day_night",
+            count="day_night",
             start_time=self.request_params.start_time,
             end_time=self.request_params.end_time,
             resolution=self.request_params.resolution,
         )
+        all_items = []
+        for item in all_accident_day_night_count:
+            at: DayNight = DayNight(item["day_night"])
+            item["day_night"] = at.get_label()
+            all_items.append(item)
+        res = sorted(all_items, key=lambda x: x["count"], reverse=True)
+        return res
+
+    def is_included(self):
+        return super().is_included()
```

Comment:
Is this needed? Doesn't it happen automatically?

Reply:
You are right, removing this